### PR TITLE
feat(eventmap-params): add destructuring options

### LINKED ISSUE #200: eventmap-params won't consider destructuring
Destructuring parameters in event-map methods will yield `eventmap-params` error. For example the following will fail:

```js
Template.foo.events({
  'change input[data-action="bar"]'({ target }, { findAll }) {
    findAll('input[data-action="baz"]').forEach(input => {
      input.checked = target.checked;
    });
  },
});
```

Is this intended, and we shuldn't destructure our event-maps, or is this something that hasn't been implemented yet?

### PR DESCRIPTION
**Pull request by runarberg**
Closes #200

### CODE REVIEW

File: docs/rules/eventmap-params.md [hunk 1]
```diff
@@ -70,6 +76,54 @@ Template.foo.events({
 
 ```
 
+#### Destructuring
+
+You can optionally allow destructuring the parameters.
```
Comment (dferber90) [inline docs/rules/eventmap-params.md]: Shouldn't this be "You can optionally **forbid** destructuring the parameters."? It's allowed by default isn't it?

**Comment by dferber90** [comment]
Thank you for the PR! Great work 👍 

Can you fix the documentation? I'm happy to merge then :)

File: docs/rules/eventmap-params.md [hunk 2]
```diff
@@ -70,6 +76,54 @@ Template.foo.events({
 
 ```
 
+#### Destructuring
+
+You can optionally allow destructuring the parameters.  
+You can set `disallowDestructuring` to `"event"`, `"templateInstance"`, or `"both"`, to force no destructuring on the event parameter, template-instance parameter, or both respectively.
```
Comment (dferber90) [inline docs/rules/eventmap-params.md]: This is a nitpick, but the plugin already contains a rule which uses `prevent` as the verb to not allow something.
Can you change every occurrence of `disallow` to `prevent`?

**Comment by dferber90** [comment]
@runarberg Any updates?

**Comment by dferber90** [comment]
Thank you once again. Sorry for sitting on it, I missed the second commit.

This is now released as the `next` version.
If you `npm install eslint-plugin-meteor@next` (or as 3.6.0), you can test it before I move the latest tag to this version.

**Comment by dferber90** [comment]
I have moved the `latest` tag to this version. Works like a charm. Thank you once again 👍

**Comment by runarberg** [comment]
Your welcome, and thank you as well. 😄

**Pull request merged**